samples/star/src/main/kotlin/com/slack/circuit/star/MainActivity.kt

```diff
@@ -80,7 +82,8 @@ class MainActivity @Inject constructor(private val circuit: Circuit) : AppCompat
           val navigator = rememberAndroidScreenAwareNavigator(circuitNavigator, this::goTo)
           val windowSizeClass = calculateWindowSizeClass(this)
           CompositionLocalProvider(
-            LocalWindowWidthSizeClass provides windowSizeClass.widthSizeClass
+            LocalWindowWidthSizeClass provides windowSizeClass.widthSizeClass,
+            LocalRetainedStateRegistry provides continuityRetainedStateRegistry()
```

Comment:
Do you think we should do this by default with `CircuitCompositionLocals` now?

Reply:
Yeah the same thing crossed my mind. I can follow up and do this.

Reply:
In fact, maybe we should do a release where it's still optional and let people test it out. We can turn it on for everyone in a subsequent release. WDYT?

Reply:
That sounds good to me 👍